docs/builtins.md

```diff
@@ -654,6 +706,12 @@ err: true
 errmsg: str2num: cannot parse "NOT A NUM"
 ```
 
+If you want your own code or function to cause a recoverable error,
+follow the convention of setting the `err` variable to `true` and the
+`errmsg` variable to a message describing the error in the error case.
+In the non-error case, make sure to set the `err` variable to `false`
+and the `errmsg` variable to the empty string.
```

Comment:
The earlier paragraph on recoverable errors just says `err` is set to `false` - I think it should be the same here - just set `err` to `false` and not worry about `errmsg`. `errmsg` is only relevant when `err` is `true`.

Reply:
Done.